orchestrator/orchestrator.go

```diff
@@ -202,14 +202,11 @@ func (o *Orchestrator) Endpoint(chainId uint64) string {
 
 func (o *Orchestrator) ConfigAsString() string {
 	var b strings.Builder
-
-	if o.l1Chain != nil {
-		fmt.Fprintf(&b, "L1:\n")
-		fmt.Fprintf(&b, "  %s\n", o.l1Chain.String())
-	}
+	l1Cfg := o.l1Chain.Config()
+	fmt.Fprintf(&b, "L1: Name: %s  ChainID: %d  RPC: %s  LogPath: %s\n", l1Cfg.Name, l1Cfg.ChainID, o.l1Chain.Endpoint(), o.l1Chain.LogPath())
 
 	if len(o.l2OpSims) > 0 {
-		fmt.Fprintf(&b, "L2:\n")
+		fmt.Fprintf(&b, "\nL2s: Predeploy Contracts Spec ( %s )\n", "https://specs.optimism.io/protocol/predeploys.html")
```

Comment:
maybe we should mention that the l2 contract addresses can be found in the spec?